pennylane/devices/tests/test_measurements.py

```diff
@@ -1630,6 +1631,9 @@ class MyMeasurement(SampleMeasurement):
             def process_samples(self, samples, wire_order, shot_range=None, bin_size=None):
                 return 1
 
+            def process_counts(self, counts: dict, wire_order: Wires):
+                return counts
```

Comment:
```suggestion
                return 1
```

Just makes it compatible with the `process_samples` method.